### PR TITLE
Harry potter

### PR DESCRIPTION
**Pull request by pritamkr1**
## Related Issue
  - Info about Issue or bug

Closes: #653 

### Describe the changes you've made 

Basically! used harry potter character API and fetched with js and also added the AOS CSS library.

### Describe if there is any unusual behavior of your code(Write `NA` if there isn't)
NA

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
![x] New and existing unit tests pass locally with my changes.

### Screenshots
Put any screenshot(s) of the project here.
herry

### CODE REVIEW

**Comment by pritamkr1** [comment]
@Vishal-raj-1  Please look into #748

**Comment by 991rajat** [comment]
@pritamkr1 Project not working.
Sync the main branch, your current project is way too old and squash the commits there are too many commits. Please fix this.

**Comment by pritamkr1** [comment]
@Vishal-raj-1 Please look into #748

File: script.js [hunk 1]
```diff
@@ -203,8 +203,18 @@ let projectData = [
...
+    projectName: "Progress bar",
+    projectImage: "assets/GIFs/progressbar.png",
+    projectUrl: "public/progressbar.html",
   }
-];
+  ,
+  {
+    projectName: "Herry Potter Characters",
+    projectImage: "assets/Images/herry.png",
+    projectUrl: "public/herry.html",
```
Comment (991rajat) [inline script.js:216]: Not working. Fix the name.

File: public/progressbar.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
```
Comment (991rajat) [inline public/progressbar.html:1]: Is this file related to your project?
If it is there are conflicts

**Review by 991rajat — COMMENTED** [review]
Fix the project it's not working.

File: script.js (hunk 1 above)
Comment (pritamkr1) [reply script.js:216]: can I fork again??

Comment (pritamkr1) [reply script.js:216]: please look into #1020  my laptop get crashed that's why I did .

**Comment by 991rajat** [comment]
@pritamkr1 Should I close this?
